internal/testtypes/builders.go

```diff
@@ -62,6 +66,30 @@ func (c *ClusterBuilder) WithClusterClass(class clusterv1.ClusterClass) *Cluster
 	return c
 }
 
+// WithTopologyVersion adds the passed TopologyVersion to the ClusterBuilder.
+func (c *ClusterBuilder) WithTopologyVersion(topologyVersion string) *ClusterBuilder {
+	c.topologyVersion = topologyVersion
+	return c
+}
+
+// WithMachineDeploymentTopology adds the passed MachineDeploymentTopology to the ClusterBuilder.
+func (c *ClusterBuilder) WithMachineDeploymentTopology(md []clusterv1.MachineDeploymentTopology) *ClusterBuilder {
```

Comment:
Should this be 
```suggestion
func (c *ClusterBuilder) WithTopologyMachineDeployments(md []clusterv1.MachineDeploymentTopology) *ClusterBuilder {
```
To be consistent with `WithTopologyVersion`?